### PR TITLE
refactor(errorHandler): rename to _errorHandler

### PR DESCRIPTION
**Pull request by PatrickJS**

### CODE REVIEW

File: modules/@angular/platform-browser/src/browser.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export function initDomAdapter() {
   BrowserGetTestability.init();
 }
 
-export function errorHandler(): ErrorHandler {
+export function _errorHandler(): ErrorHandler {
```
Comment (vicb) [inline modules/@angular/platform-browser/src/browser.ts]: why ? (`export` vs`_`)

Comment (gdi2290) [inline modules/@angular/platform-browser/src/browser.ts]: it's following the conventions of the other factory functions
_resolveDefaultAnimationDriver
_document

Comment (vicb) [inline modules/@angular/platform-browser/src/browser.ts]: I would have chosen the other way then !
I'll discuss with the team

**Comment by vicb** [comment]
we sometimes use `_` for private functions / methods - no need here as the fn is exported

**Pull request closed without merging**